package.json

```diff
@@ -31,9 +30,15 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
```

Comment:
I thought we did not want these as dependencies any more? Just peer dependencies?

Reply:
datadog.ts imports from aws-cdk/core and the other peer dependencies. When I would remove the regular dependecies from the package.json I would get a ton of import errors from the internal files that used any of those cdk packages. It seems like just having it as a dev dependency does not add it to the local node modules therefore the dd-cdk files cannot properly import.